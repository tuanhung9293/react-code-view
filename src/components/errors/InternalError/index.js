import React from 'react'
import styles from './InternalError.module.css'
import PrimaryButton from 'components/ui/buttons/PrimaryButton'

function InternalError() {
  return (
    <div className={styles.wrapper}>
      <h1>
        Something went wrong !!!
      </h1>
      <a href="/collection">
        <PrimaryButton>Back to Collections</PrimaryButton>
      </a>
    </div>
  )
}

export default InternalError
