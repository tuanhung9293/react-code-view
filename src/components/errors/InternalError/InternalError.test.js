import React from 'react'
import {shallow} from 'enzyme'
import InternalError from './'

describe('renders', () => {
  it('renders InternalError', () => {
    expect(shallow(
        <InternalError />
    )).toMatchSnapshot()
  })
})

