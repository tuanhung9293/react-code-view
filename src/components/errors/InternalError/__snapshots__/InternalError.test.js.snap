// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders renders InternalError 1`] = `
<div
  className="wrapper"
>
  <h1>
    Something went wrong !!!
  </h1>
  <a
    href="/collection"
  >
    <PrimaryButton>
      Back to Collections
    </PrimaryButton>
  </a>
</div>
`;
